modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/graphql/queries.js

```diff
@@ -279,9 +279,14 @@ export const getKoroneikiAccounts = gql`
 					partner
 					region
 					slaCurrent
-					slaCurrentEndDate
-					slaExpired
 					slaFuture
+					slaExpired
+					slaCurrentEndDate
+					slaCurrentStartDate
+					slaFutureEndDate
+					slaFutureStartDate
+					slaExpiredEndDate
+					slaExpiredStartDate
```

Comment:
All keys should be ordered by name ascending

Reply:
Sure!